src/Compilers/CSharp/Test/Semantic/Semantics/RecordTests.cs

```diff
@@ -15378,7 +15357,6 @@ public void Overrides_01()
                 "System.Int32 B.Y.get",
                 "void modreq(System.Runtime.CompilerServices.IsExternalInit) B.Y.init",
                 "System.Int32 B.Y { get; init; }",
-                "System.String B.ToString()",
```

Comment:
`ToString()` is no longer generated when the base implementation is sealed.

Reply:
This member should indeed be absent, both with language version 9 and preview.

Reply:
I don't get it. Why should it be absent in C# 9? Are you saying the existing test was wrong?

Reply:
Yes, the existing test was wrong given the change to the language specification.

We should always produce the same symbols and bind the same way, regardless of language version. The only thing that can differ is what diagnostics are produced.
For example if you used `record struct` in an older language version, we'll still probably produce a symbol with members. But we'll also produce a diagnostic. When you switch to a newer language version, the error merely goes away.

Regarding the present test: the `ToString` member should never be produced when the base type has a sealed `ToString`. But an error should be produced when that happens with a language version which isn't high enough. This test should remain an error in C# 9, but the reason changes (it's no longer because we *can't* handle the situation, it's because we *won't* given the insufficient language version).

Reply:
> We should always produce the same symbols and bind the same way, regardless of language version. The only thing that can differ is what diagnostics are produced.

Ah, thanks, that answers a question I was going to ask: how to detect if a feature is enabled without emitting a diagnostic if it's not. And the answer is "I don't need to"...

Reply:
Done, I think